src/wasm.h

```diff
@@ -2297,7 +2297,9 @@ class Global : public Importable {
 
 class Tag : public Importable {
 public:
-  Signature sig;
+  HeapType type;
+  Type params() { return type.getSignature().params; }
```

Comment:
```suggestion

    Type params() { return type.getSignature().params; }
```
(just to separate properties from methods, which we generally do)